### PR TITLE
Fixes after client review

### PR DESCRIPTION
**Pull request by macnablocky**

### CODE REVIEW

File: packages/apps/human-app/server/src/common/config/gateway-config.service.ts [hunk 1]
```diff
@@ -78,9 +78,6 @@ export class GatewayConfigService {
             [ReputationOracleEndpoints.ENABLE_LABELING]: {
               endpoint: '/user/register-labeler',
               method: HttpMethod.POST,
-              params: {
-                api_key: this.envConfig.hcaptchaLabelingApiKey,
```
Comment (macnablocky) [inline packages/apps/human-app/server/src/common/config/gateway-config.service.ts:82]: Not tested

File: packages/apps/human-app/server/src/common/interceptors/axios-request.interceptor.ts [hunk 2]
```diff
@@ -6,21 +6,25 @@ import { EnvironmentConfigService } from '../config/environment-config.service';
 export class AxiosRequestInterceptor {
   private readonly logger = new Logger(AxiosRequestInterceptor.name);
 
-  constructor(private configService: EnvironmentConfigService) {
```
Comment (MWBlocky) [inline packages/apps/human-app/server/src/common/interceptors/axios-request.interceptor.ts:9]: Remove unused import

File: packages/apps/human-app/server/src/common/interceptors/axios-request.interceptor.ts [hunk 3]
```diff
@@ -6,21 +6,25 @@ import { EnvironmentConfigService } from '../config/environment-config.service';
...
         const { url, method, params, data, headers } = config;
-        this.logger.log('NEW REQUEST:');
-        this.logger.log(`Request URL: ${url}`);
-        this.logger.log(`Method: ${method}`);
-        this.logger.log(`Params: ${JSON.stringify(params ?? {})}`);
-        this.logger.log(`Body: ${JSON.stringify(data ?? {})}`);
-        this.logger.log(`Headers: ${JSON.stringify(headers ?? {})}`);
+        this.logger.log(
+          `
+---------------NEW REQUEST: -----------------------------------------------------------------------------
```
Comment (MWBlocky) [inline packages/apps/human-app/server/src/common/interceptors/axios-request.interceptor.ts]: Consider using structured logging, such as JSON, for better readability and parsing in systems like Graylog

File: packages/apps/human-app/server/src/common/interceptors/axios-request.interceptor.ts [hunk 4]
```diff
@@ -6,21 +6,25 @@ import { EnvironmentConfigService } from '../config/environment-config.service';
...
+            Request URL: ${url} 
+            Method: ${method}
+            Params: ${JSON.stringify(params ?? {})}
+            Body: ${JSON.stringify(data ?? {})}
+            Headers: ${JSON.stringify(headers ?? {})}
```
Comment (MWBlocky) [inline packages/apps/human-app/server/src/common/interceptors/axios-request.interceptor.ts]: Consider adding a comment about disabling the interceptor to protect sensitive header information

File: packages/apps/human-app/server/src/common/interceptors/interceptor.module.ts [hunk 5]
```diff
@@ -1,9 +1,21 @@
...
+import { EnvironmentConfigService } from '../config/environment-config.service';
 
 @Global()
 @Module({
-  providers: [AxiosRequestInterceptor],
+  providers: [
+    {
+      provide: AxiosRequestInterceptor,
+      useFactory: (configService: EnvironmentConfigService) => {
+        if (configService.axiosRequestLoggingEnabled) {
```
Comment (MWBlocky) [inline packages/apps/human-app/server/src/common/interceptors/interceptor.module.ts:11]: Consider using configurable logging levels (e.g., debug) from environment settings for a more flexible approach

**Review by BartoszSolkaBD — APPROVED** [review]

**Pull request merged**